data/blog/2024/1-what.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 1) What
+date: 2024-03-26T16:06:04.000Z
+lastmod: 2024-07-04T12:57:33.000Z
+tags: ['ETHBerlin', '04']
+authors: ['Franzi', 'Helena']
+images: ['/static/images/2024/banner-3.png']
+---
+
+## 2) To hack at ETHBerlin04
+
```

Comment:
```suggestion
```

Reply:
This is also building up onto the title above